suite-native/module-settings/src/slice.ts

```diff
@@ -41,6 +44,9 @@ export const appSettingsSlice = createSlice({
         setBitcoinUnits: (state, { payload }: PayloadAction<PROTO.AmountUnit>) => {
             state.bitcoinUnits = payload;
         },
+        setRememberModeOfferChoiceTimestamp: state => {
+            state.rememberModeOfferChoiceTimestamp = new Date().getTime();
```

Comment:
Keep reducer functions pure please, you can use `prepare` or pass it from the component...

Reply:
73ad19a3b8